gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -447,6 +461,14 @@ def run(argv=None):
   metrics_util.log_all_counters(result)
 
 
+class FusionBreak(beam.PTransform):
```

Comment:
Could you add a comment/doc here? This hack wouldn't make sense to a person without context.